src/cts/include/cts/TritonCTS.h

```diff
@@ -42,6 +42,8 @@
 #include <unordered_set>
 #include <vector>
 
+#include "Clock.h"
```

Comment:
Clock.h is in src and can't be part of the public API in this header.  Perhaps you can use forward decls instead.

Reply:
Forward decl works for "class Clock" but not for "class Clock::SubNet".    Perhaps I should pull out SubNet from Clock?  Do you have other suggestions?

Reply:
I would pull it out as it is used all over the place.  I don't see a strong reason for it to be an inner class.

Reply:
Done.  Created a new derived class ClockSubNet.